src/modules/apps/list.js

```diff
@@ -29,14 +28,8 @@ export default {
   handler: () => {
     log.debug('Starting to list apps')
     return Promise.all([
-      appEngineClient().listApps(
-        getAccount(),
-        getWorkspace()
-      ).then(prop('data')),
-      appEngineClient().listLinks(
-        getAccount(),
-        getWorkspace()
-      ).then((linkedApps) => linkedApps.map(parseLocator)),
+      apps().listApps().then(prop('data')),
+      apps().listLinks().then((linkedApps) => linkedApps.map(parseLocator)),
```

Comment:
Here we could `apps()` once and destructure the functions out of it:
`const {listApps, listLinks} = apps()`